xikolo-ios/ViewControllers/Courses/CourseDecisionViewController.swift

```diff
@@ -14,7 +14,8 @@ class CourseDecisionViewController: UIViewController {
     enum CourseContent : Int {
         case learnings = 0
         case discussions = 1
-        case courseDetails = 2
+        case announcements = 2
+        case courseDetails = 3
```

Comment:
Please the order `courseDetails`, `announcements`. similar to the web app.

Reply:
I'm not sure about this one. I think most users never visit the course details page again after enrolling. Thus, I'd prefer all the relevant content at the top. It also wouldn't be the first time we do things differently in the app. For the platform-events we also have to decide for a position ourselves because it's not present on course level in the web.